modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/form.js

```diff
@@ -37,6 +39,10 @@ AUI.add(
 			return defaultAcceptFiles(val, node, ruleValue);
 		};
 
+		var email = function (val, _node, _ruleValue) {
+			return REGEX_EMAIL && REGEX_EMAIL.test(val);
+		};
```

Comment:
Couple of minor SF nits:
- we don't need `_node` and `_ruleValue` arguments, they are unused
- there is no need for `REGEX_EMAIL` truthy check, there is no case where it can be falsy.

Reply:
These are not blockers. If you feel that the code is better as it is, feel free to forward.